mirar/pipelines/winter/generator.py

```diff
@@ -29,20 +29,42 @@
 from mirar.processors.photcal import PhotCalibrator
 from mirar.processors.split import SUB_ID_KEY
 from mirar.references.local import RefFromPath
-from mirar.references.ukirt import UKIRTRef
+from mirar.references.wfcam.wfcam_query import UKIRTOnlineQuery
+from mirar.references.wfcam.wfcam_stack import WFCAMStackedRef
 
 logger = logging.getLogger(__name__)
 
 
-def winter_reference_image_resampler(**kwargs) -> Swarp:
+def winter_reference_image_resampler_for_zogy(**kwargs) -> Swarp:
     """
     Generates a resampler for reference images
 
     :param kwargs: kwargs
     :return: Swarp processor
     """
     logger.debug(kwargs)
-    return Swarp(swarp_config_path=swarp_config_path, cache=False, **kwargs)
+    return Swarp(
+        swarp_config_path=swarp_config_path, cache=False, subtract_bkg=False, **kwargs
+    )
+
+
+def winter_wfau_component_image_stacker(**kwargs) -> Swarp:
+    """
+    Generates a resampler for reference images
+
+    :param kwargs: kwargs
+    :return: Swarp processor
+    """
+    logger.debug(kwargs)
```

Comment:
this is a bad logger message. If you want it please add some text explaining what you are printing e.g `wfau kwargs: {kwargs}`